exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/decoder/Packet.java

```diff
@@ -408,7 +408,7 @@ private void decodePcapHeader(final byte[] header, final boolean byteOrder, fina
     timestamp = timestampMicro / 1000L;
     originalLength = getIntFileOrder(byteOrder, header, offset + PacketConstants.ORIGINAL_LENGTH_OFFSET);
     packetLength = getIntFileOrder(byteOrder, header, offset + PacketConstants.ACTUAL_LENGTH_OFFSET);
-    Preconditions.checkState(originalLength < maxLength,
+    Preconditions.checkState(originalLength <= maxLength,
```

Comment:
based on my understanding from this [doc](https://wiki.wireshark.org/Development/LibpcapFileFormat#Global_Header), `maxLength` here is the `snaplen` in global header ?

Reply:
I'm honestly not sure, but I had a bunch of pcap from our honeypot that had a length of 96 and they crashed Drill.  In those instances, `maxLength` was set to 96.

Reply:
Can you please share on of the file with me ? You can email me at sorabh@apache.org.

Reply:
Hi @sohami , unfortunately, the PCAPs in question are from my company's network, so I can't share them.

Reply:
Ok. based on [this it is snaplen](https://github.com/apache/drill/blob/master/exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/decoder/PacketDecoder.java#L91) so changes are fine.